hedera-node/test-clients/src/main/resources/contract/contracts/HRC632Contract/HederaAccountService.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Did this actually compile?  The abi seems to be empty.

Reply:
Yes, and the contracts which _import_ `HederaAccountService.sol` work fine.  I _think_ (but I'm not sure) that this is because this is an _abstract_ contract - in a sense it has no ABI since it can't be instantiated, only used as a base for another contact, and there's no "sharing" of code - that derived contract's binary needs to have all the methods generated right there so that's where the ABI is ...